content/projects/naomi.md

```diff
@@ -0,0 +1,18 @@
++++
+date = "2020-02-03"
+title = "Naomi"
++++
+
+[https://naomi.unaids.org](https://naomi.unaids.org/)
+
+A web interface for a model estimating various HIV epidemic indicators at a regional level. For use by 
+countries in association with UNAIDS. The app is comprised of:
```

Comment:
Can we say "this was used in workshops with UNAIDS in December 2019-January 2020, with ongoing direct use by countries" to give a better indication of the use.  Do we want to get Jeff to provide us with a 2-sentence blurb about the science too?